client/src/app/site/deployment-center/provider-dashboards/vso-Dashboard/vso-dashboard.component.ts

```diff
@@ -127,6 +127,7 @@ export class VsoDashboardComponent implements OnChanges, OnDestroy {
           this._translateService.instant(PortalResources.disconnectingDeployment),
           this._translateService.instant(PortalResources.disconnectingDeployment)
         )
+        .take(1)
```

Comment:
How does this fix the issue

Reply:
Root cause of this bug is that the startNotification observable above emits a value when a new notification is created. So this happens when user tries re-configuring the CD. This observable will emit a value with details of new notification. And later we will update the notification with successfully disconnected description.
To ignore such emitted values, we will only take the first emitted value.
Same fix exists in kudu dashboard [here](https://github.com/vinodkumar3/azure-functions-ux/blob/dc1323a2c75d80b99088ceef04d974743ec89f64/client/src/app/site/deployment-center/provider-dashboards/kudu-dashboard/kudu-dashboard.component.ts#L312).